src/main/java/minicraft/util/resource/reloader/SyncResourceReloader.java

```diff
@@ -0,0 +1,15 @@
+package minicraft.util.resource.reloader;
```

Comment:
I think all the implementations of this class could be managed in this file.

Reply:
What if we want to maximally resolve previous version resource packs? Probably separating them is a better idea.